packages/framework/src/Model/Product/ProductOnCurrentDomainElasticFacade.php

```diff
@@ -387,7 +271,7 @@ public function getProductsCountOnCurrentDomain(): int
     public function getProductsByCategory(Category $category, int $limit, int $offset, string $orderingModeId): array
```

Comment:
This one catch my eye. 

We have 

```php
public function getPaginatedProductsInCategory(
        ProductFilterData $productFilterData,
        string $orderingModeId,
        int $page,
        int $limit,
        int $categoryId
    ): PaginationResult
```
 and
 
```php
public function getProductsByCategory(
    Category $category, 
    int $limit, 
    int $offset, 
    string $orderingModeId
): array
```

those methods look really similar to me (get me a slice of products from a category) and I'm wondering whether it can be somehow rewritten

Reply:
Yeah, getProductsByCategory will be removed in next US and moved to FE API